utils/ALA_dialogues.cfg

```diff
@@ -241,7 +241,7 @@
     [/message]
     [message]
         speaker=Divaid
-        message= _ "Because Dadoc will escape when he sees his henchmen fail to stop us in a short time; he's a very cautious guy. We have to find him quickly, otherwise we can also say him good-bye forever."
+        message= _ "Because Dadoc will escape when he sees his henchmen fail to stop us in a short time. He's a very cautious guy. We have to find him quickly, otherwise we can also say him good-bye forever."
```

Comment:
semicolon is fine here; please leave it

Reply:
It could be.

Reply:
...please leave it as in, please modify this pull request so it no longer includes this change.

Reply:
nvm I did it myself in [79b8db9](https://github.com/cooljeanius/A_Little_Adventure/pull/23/commits/79b8db9b2a888bb1d1627fd207b285db6bf07928)

Reply:
OK